.github/workflows/linux-eic-shell.yml

```diff
@@ -207,6 +219,30 @@ jobs:
       env:
         GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  check-geometry-configs:
+    runs-on: ubuntu-latest
+    needs:
+      - build-gcc
+      - list-detector-configs
+    strategy:
+      matrix:
+        detector_config: ${{ fromJson(needs.list-detector-configs.outputs.configs) }}
```

Comment:
I think this probably only works in dispatched workflows, not inside the same workflow. I have to look back at the other times I've used this.

Reply:
Should we just write a for-loop in bash, for now?

Reply:
Or provide the list manually?

Reply:
Yes that would be the easy way out. But I'd like to reuse the list for spinning up parallel geoviewer jobs, and for parallel geometry checking jobs without fail-fast. The serial model surfaces one error at a time. The parallel model surfaces them all at once.

Anyway, if it takes me more than 10 minutes to figure this out when I'm at a computer, then I'm not going to bother and just use the bash for loop.